### DISCUSSION TITLE
Unable to publish config file

### DISCUSSION THREAD

**Discussion by littpi-zz** [spatie/laravel-sitemap#384] [Q&A]
# Environment
* PHP 8.0.3
* Laravel 8.33.1
* laravel-sitemap 6.0.2

# Problem
I got an error with message *Unable to locate publishable resources.* after I tried to publish config file by inputting a command ```php artisan vendor:publish --provider="Spatie\Sitemap\SitemapServiceProvider" --tag=config```. No config file is found in the **config** directory.

# Expected Result
Config file should be generated in **config** directory without error message.

# Screenshot
<img width="882" alt="螢幕截圖 2021-04-15 下午6 37 33" src="https://user-images.githubusercontent.com/17842734/114856729-05e27e80-9e1a-11eb-8626-e510ecbc9243.png">
<img width="1308" alt="螢幕截圖 2021-04-15 下午6 39 25" src="https://user-images.githubusercontent.com/17842734/114856743-09760580-9e1a-11eb-8f46-c43bb6a0bddd.png">

**Comment by khandakerashik** [comment]
try to publish without tag.

**Reply by littpi-zz** [reply]
Oh it works now. Thanks.

btw I think it is still a bug since it will copy both config and the view files. I think it is not necessary for the view files to be published...?

**Reply by andrzejkupczyk** [reply]
Luckily there are some helpful instructions in the README.md file 😉 

https://github.com/spatie/laravel-sitemap/blob/master/README.md?plain=1#L118
